src/os/bluestore/BlockDevice.cc

```diff
@@ -84,16 +85,22 @@ void IOContext::release_running_aios()
 }
 
 BlockDevice *BlockDevice::create(CephContext* cct, const string& path,
-				 aio_callback_t cb, void *cbpriv, aio_callback_t d_cb, void *d_cbpriv)
+                                 aio_callback_t cb, void *cbpriv, 
+                                 aio_callback_t d_cb, void *d_cbpriv)
 {
   string type = "kernel";
   char buf[PATH_MAX + 1];
   int r = ::readlink(path.c_str(), buf, sizeof(buf) - 1);
+
+  dout(1) << __func__ << " path " << path << " type " << type << dendl;
```

Comment:
why is this relevant to "FreeBSD make `ceph-osd mkfs` use AIO" ?

Reply:
@tchaikov 
Because it is good to know what devices bluestore is attaching too, certainly during debuging, but also durign regular use.

Reply:
@wjwithagen i suggest split this change into smaller commits, to avoid mixing the changes changing existing logging messages with the change introducing POSIX aio support.